tko.io/src/playground/styles.css

```
@@ -0,0 +1,272 @@
+  cursor: pointer;
+}
+
+.console-clear:hover { color: var(--pg-text); }
+
+.console-messages { padding: 0.25rem 0; }
+
+.console-msg {
+  padding: 0.2rem 0.75rem;
+  color: var(--pg-text-muted);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)